common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,126 @@
+"""OLI analytics service event tracker backend."""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from django.contrib.auth.models import User
+from student.models import anonymous_id_for_user
+from track.backends import BaseBackend
+from requests_oauthlib import OAuth1Session
+
+log = logging.getLogger(__name__)
```

Comment:
@kluo I think both styles are used throughout the codebase, though I'm pretty sure this style violates pylint.

I don't think this change needs to be influenced by the (incorrect) convention elsewhere. Unless there's something I'm not seeing, this should just be a local variable.